src/app/components/applications/SSV/MyAccount/components/Validator/FundingNewValidator/FundingNewValidator.tsx

```diff
@@ -108,10 +108,11 @@ const FundingNewValidator = () => {
     if (Number(depositSSV) > 0) return <Typography className={classes.PositiveDays}>(+{depositSSV} SSV)</Typography>;
   };
 
-  const moveToNextPage = () => {
+  const moveToNextPage = async () => {
```

Comment:
there is no await, are you sure this function needs to be async

Reply:
+